WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m

```diff
@@ -635,10 +643,38 @@ - (UITableViewCell *)configureTaxonomyCellForIndexPath:(NSIndexPath *)indexPath
     return cell;
 }
 
+- (void)configureMetaSectionRows
+{
+    NSMutableArray *metaRows = [[NSMutableArray alloc] init];
+
+    if (self.apost.isMultiAuthorBlog) {
+        [metaRows addObject:@(PostMetaRowAuthor)];
+    }
+
+    [metaRows addObjectsFromArray:@[ @(PostMetaRowPublishDate),
+                                      @(PostMetaRowPublishStatus),
+                                      @(PostMetaRowVisibility) ]];
+
+    if (self.apost.password) {
+        [metaRows addObject:@(PostMetaRowPassword)];
+    }
+
+    self.postMetaSectionRows = [metaRows copy];
+}
```

Comment:
This approach looks great! The app would probably benefit from this approach in other places as well :D